services/QuillLMS/spec/services/vitally_integration/serialize_vitally_sales_account_spec.rb

```diff
@@ -22,11 +22,15 @@
   let(:subscription) { create(:subscription, account_type: Subscription::SCHOOL_PAID) }
 
   before do
+    create(:activity_classification, key: 'evidence')
```

Comment:
I'm pretty sure we have a convenience factory for this and that you can just call `create(:evidence)`